src/cmd/singularity/cli/keys_push.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func init() {
 	KeysPushCmd.Flags().SetInterspersed(false)
-	KeysPushCmd.Flags().StringVarP(&url, "url", "u", "", "overwrite the default remote url")
+	KeysPushCmd.Flags().StringVarP(&keyServerURL, "url", "u", "", "specify the key server URL (default: https://keys.sylabs.io)")
```

Comment:
Should the `push` also read key server default from config file like `sign` and `verify`?

Reply:
This is removed completely see my comment above.